kinto_emailer/__init__.py

```diff
@@ -1,9 +1,15 @@
+import re
+
 from kinto.core.events import AfterResourceChanged
 from pyramid.settings import asbool
 from pyramid_mailer import get_mailer
 from pyramid_mailer.message import Message
 
 
+EMAIL_REGEXP = re.compile(r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)")
```

Comment:
Unfortunately email regexp are really complex and this one probably doesn't covers all the cases. But we can start with that and improve it overtime.